hub/auto/util.py

```diff
@@ -6,6 +6,7 @@
 IGNORE_EXTS = [".DS_Store"]
 IMAGE_EXTS = [".jpg", ".png", ".jpeg"]
 CSV_EXTS = [".csv"]
+AUDIO_EXTS = [".aac", ".au", ".flac", ".m4a", ".mp3", ".ogg", ".wav"]
```

Comment:
can you write 1 test per audio extension?

Reply:
Sure!